src/pages/Host/HostAmenity/HostAmenity.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import axios from 'axios';
+import HostAmenityCom from './HostAmenityCom';
+
+function HostRadio() {
+  const [data, setData] = useState({
+    productList: [],
+  });
+
+  const getData = useCallback(async () => {
+    try {
+      const response = await axios.get('/data/HostImg.json');
+      console.log(response);
+      setData({ productList: response.data });
+    } catch (error) {
+      console.error(error);
+    }
+  });
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <>
+      <FormBoxAmenity>
+        <FormTextAmenity>
+          <FormTextLeft>
+            <FormBoxTitle>편의시설</FormBoxTitle>
+            <FormBoxTitleOption>*</FormBoxTitleOption>
+          </FormTextLeft>
+          <FormBoxOption>구비된 편의시설을 선택해주세요.</FormBoxOption>
+        </FormTextAmenity>
+
+        <FormIconWrap>
+          {data.productList.length > 0 &&
+            data.productList.map(iconImg => {
```

Comment:
- 어차피 길이가 0인 배열에 대해서는 map 함수가 순회를 하지 않기 때문에, 여기서의 조건부는 불필요해 보입니다!
```suggestion
          {data.productList.map(iconImg => {
```